experimental/dds/tree2/src/core/tree/anchorSet.ts

```diff
@@ -627,23 +627,6 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents>, AnchorLoca
 					}
 				}
 			},
-			// Run `withNode` on every node from the current `this.parent` to the root.
-			// Should only be called when in a field.
-			withParentNodeUpToRoot(withNode: (anchorNode: PathNode) => void) {
```

Comment:
`beforeChange` and `afterChange` events in `AnchorNode`s don't propagate to the parents anymore, that only happens in the `FlexTreeNode` layer, so this functionality became unnecessary.